data/channels.csv

```diff
@@ -10041,6 +10050,7 @@ KALBDT3.us,KALB-DT3,,CW,,US,US-LA,Alexandra,s/US-LA,eng,,FALSE,,,,,https://uploa
 KaleAwadiTV.et,Kale Awadi TV,,,,ET,,,c/ET,amh,,FALSE,,,,,https://i.imgur.com/reqk3BY.png
 KaleidoscopeTV.ru,Kaleidoscope TV,Калейдоскоп ТВ,,,RU,,,c/RU,rus,,FALSE,,,,http://www.kaleidoskoptv.tv/,https://i.imgur.com/Ub3gojM.png
 KalemehTV.uk,Kalemeh TV,,,,UK,,,c/IR,fas,religious,FALSE,,,,https://kalemeh.tv/,https://i.imgur.com/RsKKULI.png
+KalikaTV.in,Kalika TV,,,,IN,,,c/IN,mal,music,FALSE,,,,,https://i.imgur.com/KEZ5M4z.png

```

Comment:
```suggestion
KalikaTV.in,Kalika TV,,,,IN,,,c/IN,mal,music,FALSE,,,,,https://i.imgur.com/XWXFoJA.png
```
Removed the transparent background around the logo.

Reply:
Already have a transparent background!